code/modules/mob/living/simple_animal/hostile/terror_spiders/black.dm

```diff
@@ -0,0 +1,33 @@
+
+// --------------------------------------------------------------------------------
+// ----------------- TERROR SPIDERS: T2 BLACK TERROR ------------------------------
+// --------------------------------------------------------------------------------
+// -------------: ROLE: assassin, poisoner, DoT expert
+// -------------: AI: attacks to inject its venom, then retreats. Will inject its enemies multiple times then hang back to ensure they die.
+// -------------: SPECIAL: venom that does more damage the more of it is in you
+// -------------: TO FIGHT IT: if bitten once, retreat, get charcoal/etc treatment, and come back with a gun.
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/GradualGrinder
+// -------------: SPRITES FROM: Travelling Merchant, http://nanotrasen.se/phpBB3/memberlist.php?mode=viewprofile&u=2766
+
+/mob/living/simple_animal/hostile/poison/terror_spider/black
+	name = "black widow spider"
+	desc = "An ominous-looking spider, black as the darkest night, and with merciless yellow eyes."
+	altnames = list("Black Devil spider","Giant Black Widow spider","Shadow Terror spider")
```

Comment:
These are all capitalized, while the default name is all lowercase. Not sure if this is intentional or not. Same is true for most of the other spiders.